db/migrate/20221201073635_create_user.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :postsCounter
```

Comment:
Great job @obibaadoma 🤝
- As stated by the previous reviewer, kindly ensure your tests are passing. At the moment most of your tests are still failing. You may try to follow the previous reviewer's advice to fix them.